### PR TITLE
Fix: updated jackson-databind to 2.11.0 to block CVE-2020-25649

### PR DESCRIPTION
**Pull request by darveshsingh**
## Changes
 Version bump of jackson-databind to 2.11.0

## References: 
 [CVE-2020-25649](https://bugzilla.redhat.com/show_bug.cgi?id=1887664)

### CODE REVIEW

File: lib/build.gradle [hunk 1]
```diff
@@ -41,7 +41,7 @@ compileJava {
 }
 
 dependencies {
-    implementation 'com.fasterxml.jackson.core:jackson-databind:2.10.5.1'
```
Comment (lukeZhangMengxi) [inline lib/build.gradle:44]: I think `2.10.5.1` is already bug-free according to https://nvd.nist.gov/vuln/detail/CVE-2020-25649
Please note, the known affected version is "Up to (excluding) 2.10.5.1"

Comment (darveshsingh) [reply lib/build.gradle:44]: Hi Luke,
Can you please refer below links:

1. https://bugzilla.redhat.com/show_bug.cgi?id=1887664
2. https://github.com/FasterXML/jackson-databind/commit/e588f0af61b18576779ffb95a2a689a5eb1f9d15

Comment (lukeZhangMengxi) [reply lib/build.gradle:44]: Hi,
Yeah, I guess 2.11.0 might be a good one. Unfortunately, I am not from Auth0. Just want them to fix this issue as soon as possible~

**Comment by jimmyjames** [comment]
Thanks for the PR! However, the fix for that CVE [is included](https://github.com/FasterXML/jackson-databind/issues/2589#issuecomment-736973925) in `2.10.5.1`. Looking at the decompiled classes in the `2.10.5.1` jar also shows that the [fix](https://github.com/FasterXML/jackson-databind/commit/612f971b78c60202e9cd75a299050c8f2d724a59) is included there.

I'm not opposed to updating our version of Jackson in general, but I'd probably wait for the first patch release of the `2.12` stream (or at least take the latest patch of `2.11`), and look at other dependency updates at the same time.

**Comment by lbalmaceda** [comment]
@darveshsingh 2.12 is out. Do you want to update the PR with that change?

**Comment by darveshsingh** [comment]
> @darveshsingh 2.12 is out. Do you want to update the PR with that change?

Yes

**Comment by darveshsingh** [comment]
> @darveshsingh 2.12 is out. Do you want to update the PR with that change?

Hi @lbalmaceda ,

Hope you are doing good.!
Just a request/reminder to please approve the PR. :)

**Review by lbalmaceda — APPROVED** [review]

**Pull request merged**